CHANGES

```diff
@@ -6,6 +6,11 @@ arpack-ng - 3.6.0
    Example of   C/Fortran binding can be found in the TESTS/iso_c_binding_c.c     file.
    Example of C++/Fortran binding can be found in the TESTS/iso_c_binding_cpp.cpp file.
 
+ [ Franck Houssen ]
+ * Add support for ISO_C_BINDING (Fortran std 2003) to improve portability (Fortran/C/C++).
```

Comment:
?

Reply:
I know git well !... But so much not github (PR/fork and so on). These are my first steps actually with github and I have to say I am not yet really comfortable with it.... Sorry about that.

After you merged my PR, I had to create an upstream remote, merge my clone with upstream, and push force to github: this made history messy (merged into 90fba75 - this is what you see).

The only thing you may be interested in is 71acc79.

Reply:
I reseted-hard/rebased branches of my clone: pushed force back to github.
Should be clearer now... Sorry, really not used to github yet: you may look at 17717b.